integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/preaaggregate/PreAggregateListeners.scala

```diff
@@ -111,6 +113,29 @@ trait CommitHelper {
     }
   }
 
+  protected def mergeTableStatusContents(uuidTableStatusPath: String,
```

Comment:
It seems duplicate, please try to use same method from other places as well.

Reply:
done